plugins/action_trim_responses/src/trimResponses.test.ts

```diff
@@ -9,23 +9,23 @@ beforeAll(async (done) => {
   // Duplicate application responses, with some modifications
   await DBConnect.query({
     text: `
-  INSERT INTO "public".application_response (id, template_element_id, application_id, "value", is_valid, time_updated) VALUES (DEFAULT, 1009, 1000, '{"text": "Manufacturer B", "optionIndex": 1}', NULL, 'NOW()');
-  INSERT INTO "public".application_response (id, template_element_id, application_id, "value", is_valid, time_updated) VALUES (DEFAULT, 1007, 1000, '{"text": "Manufacturer", "optionIndex": 1}', NULL, 'NOW()');
-  INSERT INTO "public".application_response (id, template_element_id, application_id, "value", is_valid, time_updated) VALUES (DEFAULT, 1006, 1000, '{"text": "12345678"}', NULL, 'NOW()');
-  INSERT INTO "public".application_response (id, template_element_id, application_id, "value", is_valid, time_updated) VALUES (DEFAULT, 1005, 1000, '{"text": "craig@sussol.net"}', NULL, 'NOW()');
-  INSERT INTO "public".application_response (id, template_element_id, application_id, "value", is_valid, time_updated) VALUES (DEFAULT, 1004, 1000, '{"text": "craig_drown"}', NULL, 'NOW()');
-  INSERT INTO "public".application_response (id, template_element_id, application_id, "value", is_valid, time_updated) VALUES (DEFAULT, 1002, 1000, '{"text": "Drown"}', NULL, 'NOW()');
+  INSERT INTO "public".application_response (id, template_element_id, application_id, "value", is_valid, time_submitted) VALUES (DEFAULT, 1009, 1000, '{"text": "Manufacturer B", "optionIndex": 1}', NULL, 'NOW()');
+  INSERT INTO "public".application_response (id, template_element_id, application_id, "value", is_valid, time_submitted) VALUES (DEFAULT, 1007, 1000, '{"text": "Manufacturer", "optionIndex": 1}', NULL, 'NOW()');
+  INSERT INTO "public".application_response (id, template_element_id, application_id, "value", is_valid, time_submitted) VALUES (DEFAULT, 1006, 1000, '{"text": "12345678"}', NULL, 'NOW()');
+  INSERT INTO "public".application_response (id, template_element_id, application_id, "value", is_valid, time_submitted) VALUES (DEFAULT, 1005, 1000, '{"text": "craig@sussol.net"}', NULL, 'NOW()');
+  INSERT INTO "public".application_response (id, template_element_id, application_id, "value", is_valid, time_submitted) VALUES (DEFAULT, 1004, 1000, '{"text": "craig_drown"}', NULL, 'NOW()');
+  INSERT INTO "public".application_response (id, template_element_id, application_id, "value", is_valid, time_submitted) VALUES (DEFAULT, 1002, 1000, '{"text": "Drown"}', NULL, 'NOW()');
```

Comment:
The IDs are causing a failure -- they're referring to IDs that no longer exist. 🤷‍♂️

Reply:
Yes, I checked this now and there isn't an application 1000 for template "03_tempalte_A_featuresShowcase" anymore. This needs to be updated. I'll do this in a separated PR

Reply:
Okay, I think we'll probably want to start using Snapshots for our test suites. Maybe discuss this at our next meeting?